citeproc/model.py

```diff
@@ -95,14 +95,16 @@ def render(self, *args, **kwargs):
         return self.markup(self.process(*args, **kwargs))
 
     # TODO: Locale methods
-    def get_term(self, name, form=None):
+    def get_term(self, name, form=None, fallback_locale=True, zero_padded=False):
```

Comment:
better later (now) than never (before): with addition of these options and logic becoming more evolved, IMHO it is worth adding docstring describing them.

Reply:
I agree.